gradle.properties

```
@@ -1,4 +1,4 @@
-release_version=4.0.3
+release_version=4.0.4
```

```suggestion
release_version=4.1.0
```
because we updated minor version of jackson-bom